node/tools/Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: run-node node-configs clean
+IP?=127.0.0.1:8000
+
+run-node:
+	export RUST_LOG=INFO && cargo run -- --config-file node-configs/${IP}/config.json --node-key node-configs/${IP}/node_key --validator-key node-configs/${IP}/validator_key --database ./database/${IP} 
+
```

Comment:
nit: note that if you execute binary from "node-configs/${IP}" working directory, you don't have to set any flags.